projects/native-charts-ng/micro-bar/si-micro-bar.component.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) Siemens 2016 - 2025
+ * SPDX-License-Identifier: MIT
+ */
+import { ChangeDetectionStrategy, Component, computed, input } from '@angular/core';
+
+export interface MicroBarSeries {
+  values: number[];
+  colorToken: string;
```

Comment:
those need a js-doc comment.

The color token should also have a link to the documentation of our tokens